packages/inference/src/HfInference.ts

```diff
@@ -615,6 +699,16 @@ export class HfInference {
 		return res?.[0];
 	}
 
+	/**
+	 * Use to continue text from a prompt. Same as `textGeneration` but returns stream that can be read one token at a time
+	 */
+	public async textGenerationStream(
+		args: TextGenerationArgs,
+		options?: Options
+	): Promise<ReadableStream<TextGenerationStreamReturn>> {
+		return await this.streamingRequest<TextGenerationStreamReturn>(args, options);
```

Comment:
It's really nitpicking but this `await` is useless and create unnecesary promises on the event loop